aries-site/src/examples/templates/filtering/FilteringTable/FilteringTable.js

```diff
@@ -0,0 +1,132 @@
+
+import {
+  FilterControls,
+  FiltersProvider,
+  useFilters,
+} from '../../FilterControls';
+import { servers } from './mockData';
+
+export const FilteringTable = () => {
+  // Define which attributes should be made available for the user
+  // to filter upon
+  const filtersConfig = [
+    {
+      property: 'hardware.health.summary',
+      label: 'Status',
+      filterType: 'CheckBoxGroup',
+    },
+    {
+      property: 'state.connected',
+      label: 'State',
+      filterType: 'CheckBoxGroup',
+      render: datum => (datum ? 'Connected' : 'Not Connected'),
+    },
+    {
+      property: 'hardware.powerState',
+      label: 'Power',
+      filterType: 'CheckBoxGroup',
+    },
+    {
+      property: 'hardware.model',
+      label: 'Model',
+      filterType: 'CheckBoxGroup',
+    },
+  ];
+
+  return (
+    <Box pad="small" gap="medium">
+      <Heading level={2} margin="none">
+        Servers
+      </Heading>
+      <FiltersProvider>
+        <Box gap="medium">
+          <Box align="start" direction="row" justify="between" wrap>
+            <FilterControls
+              data={servers}
+              filters={filtersConfig}
+              primaryKey="id"
+              searchFilter={{ placeholder: 'Search servers...' }}
+            />
+            <Menu
+              kind="toolbar"
+              label="Actions"
+              items={[{ label: 'Power On' }, { label: 'Power Off' }]}
+            />
+          </Box>
+          <ServerResults />
+        </Box>
+      </FiltersProvider>
+    </Box>
+  );
+};
+
+const columns = [
+  {
+    property: 'hardware.serialNumber',
+    header: 'Serial',
+  },
+  {
+    property: 'state.connected',
+    header: 'State',
+    render: datum => (datum.state.connected ? 'Connected' : 'Not Connected'),
+  },
+  {
+    property: 'hardware.powerState',
+    header: 'Power',
+  },
+  {
+    property: 'hardware.model',
+    header: 'Model',
+  },
+];
+
+const statusIcons = {
+  Warning: <StatusWarningSmall color="status-warning" size="small" />,
+  OK: <StatusGoodSmall color="status-ok" size="small" />,
+  Critical: <StatusCriticalSmall color="status-critical" size="small" />,
+  Unknown: <StatusUnknownSmall color="status-unknown" size="small" />,
+};
+
+const ServerResults = () => {
+  const size = useContext(ResponsiveContext);
+  const { filteredResults, selected, setSelected } = useFilters();
+
+  return (
+    <Box height="medium" overflow="auto">
+      <DataTable
+        data={filteredResults}
+        columns={[
+          {
+            property: 'hardware.health.summary',
+            header: 'Status',
+            render: datum =>
+              datum.hardware.health.summary
+                ? statusIcons[datum.hardware.health.summary]
+                : '-',
+            align: 'center',
+            sortable: false,
+          },
+          {
+            property: 'displayName',
+            header: 'Name',
+            pin: size === 'small',
+          },
```

Comment:
We need one of the columns to be styled as a primary identifier (i.e. bold).
![Screen Shot 2021-06-14 at 3 51 51 PM](https://user-images.githubusercontent.com/1756948/121964308-93fab880-cd28-11eb-9d43-ec7b48d3b81e.png)

Since primaryKey is "id", I think we have to have "id" included to receive primary styling.

```suggestion
          {
            property: 'id',
            header: 'Name',
            pin: size === 'small',
            render: datum => datum.displayName,
          },
```

The alternative would be to style the displayName as `<Text weight="bold">{datum.displayName}</Text>` but then we are breaking from the theme a bit.